recipe/meta.yaml

```diff
@@ -56,6 +60,31 @@ outputs:
         convenient.
 
         This package contains utility modules to talk to Avahi with Python easier.
+  - name: avahi
+    script: lib-avahi.sh
+    requirements:
+      build:
+        # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+        # Compilers are named 'c', 'cxx' and 'fortran'.
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
```

Comment:
I don't think a C++ compiler is required (or used) here.

```suggestion
```

Reply:
Autoconf fails if a C++ compiler is not found